scripts/generate_road_colors.py

```diff
@@ -0,0 +1,211 @@
+from colormath.color_conversions import convert_color
+from colormath.color_objects import LabColor, LCHabColor, SpectralColor, sRGBColor, \
+XYZColor, LCHuvColor, IPTColor
+from colormath.color_diff import delta_e_cie2000
+import numpy
```

Comment:
INSTALL.md already has "[PyYAML](http://pyyaml.org/wiki/PyYAML) if editing the MML (layer definition) (...)"


I also think that it would be better to keep it in one place to make installation a bit less frustrating.

I will create a new subsection "Development dependencies" to make it clear that it is necessary to use the style.